evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -132,7 +132,7 @@ def test_all_estimators(
         assert len((_all_estimators_used_in_search())) == 9
     else:
         if is_using_conda:
-            n_estimators = 16
```

Comment:
Was talking to @jeremyliweishih about this one.  Apparently this line isn't covered in the global checks, but my fiddling with it caused the patch diff to fail.